lib/mathlib.cpp

```diff
@@ -143,6 +143,8 @@ double MathLib::toDoubleNumber(const std::string &str)
     // nullcheck
     else if (isNullValue(str))
         return 0.0;
+    else if (isFloat(str)) // Workaround libc++ bug at http://llvm.org/bugs/show_bug.cgi?id=17782
```

Comment:
I do not think that we should start to implement workarounds for runtime library bugs.